docs/config.md

```diff
@@ -173,6 +173,9 @@ profile_models: <comma-delimited-string-list>
 # Disables automatic config search
 [ run_config_search_disable: <bool> | default: false ]
 
+# Enables profiling models concurrently
+[ run_config_profile_models_concurrently_enable: <bool> | default: false ]
```

Comment:
Do we need 'enable' for a positive option? Seems like `run_config_profile_models_concurrently` is enough

Reply:
My personal preference is that we be explicit with enable/disable. He had the same debate in HW years back (around CREGs) and this helped to prevent issues.

Reply:
It should be consistent with the rest of the options. It looks like the other run_config* options are disable/enable, so this is fine to keep.